src/m_menu.c

```diff
@@ -5737,19 +5744,34 @@ static boolean M_SetupResponder(event_t *ev, menu_action_t action, int ch)
 
 } // End of Setup Screen processing
 
+static boolean M_SaveLoadResponder(menu_action_t action, int ch);
+
 static boolean M_MainMenuMouseResponder(void)
 {
     if (messageToPrint)
     {
         return false;
     }
 
-    static boolean active_thermo;
     menuitem_t *current_item = &currentMenu->menuitems[itemOn];
+
+    if (current_item->flags & MF_PAGE)
+    {
+        if (M_InputActivated(input_menu_enter))
+        {
+            if (savepage == savepage_max)
+                savepage = -1;
+            M_SaveLoadResponder(MENU_RIGHT, 0);
```

Comment:
So, it only skips to the next page even if you click on "prev"?

Reply:
We don't have "prev", just little `<-`. I've tried it, it looks clunky.

This is one of those things I would prefer to do in a separate PR. Once the reset default is fixed, I suggest we stop working with this PR and decide to merge.